src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -116,8 +117,12 @@ class SeekBar extends Slider {
     }
```

Comment:
btw mis-indented `}` while you're here 😄

Reply:
Github just makes it look that way, it's properly indented in code (and not PRs)
https://github.com/videojs/video.js/blob/3d093ede98f2ad6204ff44c9631b0aa18bbf0cc5/src/js/control-bar/progress-control/seek-bar.js#L114-L116

Reply:
_flips table_